docs/reference/filters.md

```diff
@@ -1748,6 +1748,106 @@ foo: Path("/baz")
 Configures rate limit of 100 requests per second for each `backend1` and `backend2` and responds
 with `429 Too Many Requests` when limit is reached.
 
+## clusterLeakyBucketRatelimit
+
+Implements leaky bucket rate limit algorithm that uses Redis as a storage.
+Requires command line flags `-enable-ratelimits`, `-enable-swarm` and `-swarm-redis-urls` to be set.
+
+The leaky bucket is an algorithm based on an analogy of how a bucket with a constant leak will overflow if either
+the average rate at which water is poured in exceeds the rate at which the bucket leaks or if more water than
+the capacity of the bucket is poured in all at once, see https://en.wikipedia.org/wiki/Leaky_bucket
+
+Parameters:
+
+* label (string)
+* leak rate volume (int)
+* leak rate period (time.Duration)
+* capacity (int)
+* increment (int)
+
+The bucket label, leak rate (volume/period) and capacity uniquely identify the bucket.
+
+Label supports [template placeholders](#template-placeholders).
```

Comment:
Can we make this explicit field rather than implicit in the "group" as we have "group" used for redis key in other `cluster*Ratelimit`?
I think we should stay with the same user interface for all the kinds of ratelimits if we can.

Reply:
The filter already has quite a lot of parameters, we may also add response code customization later on like in others.
The parameters are leaky-bucket algorithm specific anyway and I don't see a reasonable meaning for a group. I think group was there before we had template placeholders to enable dynamic ratelimit keys. Today it is easier to either have a const value like `"my-api-limit"` or a dynamic value like `"auth-${request.header.Authorization}"` both via a single parameter.

Reply:
I got it thanks